src/organisms/announcementCard/styles.tsx

```diff
@@ -0,0 +1,32 @@
+import { Styled } from '../../theme';
+import { AnnouncementCardProps } from './types';
+
+export const AnnouncementCardContainer = Styled.div<Pick<AnnouncementCardProps, 'width'>>`
+    background: ${({ theme }) => theme.containerAndCardShades.BG_SHADE_PLUS_4};
+    box-shadow: ${({ theme }) => theme.dropShadow.REGULAR};
+    width: ${({ width }) => width};
```

Comment:
@ArinCantCode  card fully locked for a specific width, i think this page , just 100% support mobile view
![Screen Shot 2022-06-19 at 10 16 12](https://user-images.githubusercontent.com/35912161/174470165-944518bc-b586-4155-9dbb-4c78ff1b7772.png)

Reply:
Can you clarify? Should 100% be a default value?

Reply:
@ArinCantCode yep, u gonna control the width into the page with a breakpoints